### PR TITLE
[ADD] module mrp_produce_uos

### PR DESCRIPTION
**Pull request by tafaRU**
This module adds the field uos quantity to Product Produce wizard allowing the user to use the uos quantity (instead of the standard uom) to specify the quantity to be produced.

### CODE REVIEW

**Comment by eLBati** [comment]
:+1: thanks

**Comment by archetipo** [comment]
:+1:

**Comment by tafaRU** [comment]
Could someone of @OCA/manufacturing-maintainers merge it?

Thank you in advance.

**Comment by tafaRU** [comment]
Could someone of @OCA/manufacturing-maintainers please review and possibly merge this PR?

Thank you in advance!

**Comment by tafaRU** [comment]
@pedrobaeza @max3903 coul you please please review and possibly merge this PR?

Thank you in advance!

File: mrp_produce_uos/models/mrp_production.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        mrp_production = self.browse(production_id)
+
+        for move in mrp_production.move_created_ids2:
+            product_uom_qty = move.product_uom_qty
+            p_qty = mrp_production.product_qty
+            p_uos_qty = mrp_production.product_uos_qty
+            product_uos_qty = p_uos_qty * (product_uom_qty / p_qty)
+            # I used sql to avoid
+            # https://github.com/odoo/odoo/blob/8.0/addons/stock/stock.py#L1980
+            self.env.cr.execute(
```
Comment (pedrobaeza) [inline mrp_produce_uos/models/mrp_production.py]: You can still use directly `models.write()` for not avoiding security rules or the recalculation of computed fields.

Comment (tafaRU) [inline mrp_produce_uos/models/mrp_production.py]: The link in the meantime has been modified and got no clues on the new one.
I've tried but I remember it causes a loop. What I did seems to be the only way.

Comment (pedrobaeza) [inline mrp_produce_uos/models/mrp_production.py]: Try what I propose that I think it will work for sure.

Comment (tafaRU) [inline mrp_produce_uos/models/mrp_production.py]: Tried with:
`move.sudo().write({'product_uos_qty': product_uos_qty})
`

but I get the following error: https://github.com/odoo/odoo/blob/8.0/addons/stock/stock.py#L2008

Any hints how to fix it?

**Comment by tafaRU** [comment]
@pedrobaeza, I've addressed your last remark with https://github.com/OCA/manufacture/commit/e8829848191717c4f265c2ff512bb2840d387477 :wink: 

Thanks a lot for your review! I think now it should be mergeable, what do you think about it?

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**